### DISCUSSION TITLE
Streaming like in Guilded

### DISCUSSION THREAD

**Discussion by jamescooperation** [stoatchat/discussions#742] [Feature Suggestions]
so guilded has a streaming feature and it is supposed to display whatever you are showing (either your entire screen, or just an application)

this is what it looks like
![Screenshot 2024-06-04 163831](https://github.com/revoltchat/revolt/assets/171699098/5a25d877-1dc9-48b4-9be8-fafcc7cfef5f)

**Comment by randomy1** [comment]
This will be added, but not right now. (according to the revolt devs/roadmap) 

itll probably be added in like a month or so (idk)

**Reply by bmvoss123** [reply]
I would assume this is part of the Tauri app rework in [Issue #14](https://github.com/revoltchat/frontend/issues/14).

**Comment by randomy1** [comment]
and no, im not a revolt dev. im just stating this so people know, ive seen like 2 suggestions abt this

**Reply by jamescooperation** [reply]
k
